assets/constants.js

```diff
@@ -474,5 +474,37 @@ var constants = {
 
   "USE_NEW_SUGGESTION_FRAMEWORK": false,
 
-  "ENABLE_NEW_STRUCTURES": false
+  "ENABLE_NEW_STRUCTURES": false,
+
+  "NEW_STATE_TEMPLATE": {
+      "classifier_model_id": null,
+      "content": {
+          "html": "",
+          "content_id": "content"
+      },
+      "interaction": {
+            "id": null,
+            "customization_args": {},
+            "answer_groups": [],
+            "default_outcome": {
+                "dest": "Introduction",
```

Comment:
Does this get substituted? If so maybe add a comment and have the default be null?